sdk/ml/azure-ai-ml/azure/ai/ml/_schema/queue_settings.py

```diff
@@ -0,0 +1,19 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+# pylint: disable=unused-argument,no-self-use
+
+from marshmallow import fields, post_load
+
+from azure.ai.ml._schema.core.schema_meta import PatchedSchemaMeta
+
+
+class QueueSettingsSchema(metaclass=PatchedSchemaMeta):
```

Comment:
@vivijay91 I am working on a [PR to support queue_settings](https://github.com/Azure/azure-sdk-for-python/pull/29122). Looks like we need to join efforts here. Thanks!